### ISSUE TITLE
How to change the size of clockpicker?

### ISSUE THREAD

**Issue by info170** [weareoutman/clockpicker#135]
I need to place it on center of screen and change the size to 80% of screen.

**Comment by djibe** [comment]
Are you sure Clockpicker is the best answer to your problem ?

**Comment by info170** [comment]
Yes, Clockpicker is convenient tool but the reason is that it used on tablet and it is too small to tap by finger. So, I want to increase the size. I found how to change position but can't change size.

**Comment by djibe** [comment]
Yep it won't be fixed before long.
So you should move to another plugin.

**Comment by info170** [comment]
Thank you for a quick answer. Will find another way.
